src/content/docs/reference/recipe.mdx

```diff
@@ -3,7 +3,23 @@ title: recipe.yml
 description: A reference page in my new Starlight docs site.
 ---
 
-https://diataxis.fr/reference/
+## name
 
-- technical description of recipe
-- list of all keys
\ No newline at end of file
+The image name. Used when publishing to GHCR as `ghcr.io/<user>/<name>`
+
+## description
+
+The image description. Published to GHCR in the image metadata
+
+## base-image
+
+The image upon which all your customizations will be based. It can be any of the images listed on the [uBlue website](https://universal-blue.org/images/) or any BlueBuild-produced image.
+
+Example:
+```
+base-image: ghcr.io/ublue-os/silverblue-main
+```
+
+## image-version
+
+The version of the base image to be used. Tag options include the the `latest` tag and the tag corresponding to the latest Fedora release. `latest` will change over to the next Fedora release as soon as a base image is available for that version. A specific Fedora release tag will keep the image pinned to that Fedora release until you choose to modify it. A list of all available tags can be viewed by pasting your `base-image` url into your browser.
```

Comment:
The _tag_ of the base image to build on. Used to select version explicitly (`39`) or to always use the latest stable version (`latest`). Other content alright.